config/kcp/apibinding_integration.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIBinding
+metadata:
+  name: integration-binding
+spec:
+  reference:
+    workspace:
+      path: root:users:xr:vd:rh-sso-<your_user_account>:test-workspace
+      exportName: integration
+  permissionClaims:
+    - group: ""
+      resource: "secrets"
+      state: Accepted
+    - group: ""
+      resource: "namespaces"
+      state: Accepted
+    - group: ""
+      resource: "configmaps"
+      state: Accepted
+    - resource: "applications"
+      group: "appstudio.redhat.com"
+      identityHash: null # has apiexport
```

Comment:
the has and gitops export have combined. See https://github.com/redhat-appstudio/application-api/blob/main/config/kcp/apiexport.yaml

Reply:
@johnbieren Thank you! Let me update my code.